add-column.md

```diff
@@ -18,6 +18,121 @@ The user must have the `CREATE` [privilege](privileges.html) on the table.
 
 ## Parameters
 
+| Parameter | Description |
+|-----------|-------------|
+| `table_name` | The name of the table to which you want to add the column. |
+| `name` | The name of the column you want to add. The column name must follow these [identifier rules](keywords-and-identifiers.html#identifiers) and must be unique within the table but can have the same name as indexes or constraints.  |
+| `typename` | The [Data type](data-types.html) of the new column. |
```

Comment:
Can you make `data type` lowercase? Sorry, should have asked for that in my last comment.